docs/pom.xml

```diff
@@ -27,7 +39,43 @@
         </dependency>
         <dependency>
             <groupId>io.quarkiverse.azureservices</groupId>
-            <artifactId>quarkus-azure-storage-blob-deployment</artifactId>
+            <artifactId>quarkus-azure-http-client-vertx-deployment</artifactId>
```

Comment:
These are internal libraries. Could you explain why we include them in the documentation POM?

Reply:
Good catch. These internal libraries needn't to be explicitly specified as dependencies as they've already been dependencies for external extensions. Fixed in 7daaf6c434e7810823ed69ec411e67f170b50e49.